### PR TITLE
add dbaas example

### PR DESCRIPTION
**Pull request by PanLiyuan**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

File: website/docs/r/resource_instance.html.markdown [hunk 1]
```diff
@@ -35,6 +35,33 @@ resource "ibm_resource_instance" "resource_instance" {
...
+  service = "hyperp-dbaas-postgresql"
+  plan = "postgresql-free"
+  location = "us-south"
+  resource_group_id = data.ibm_resource_group.group.id
+  timeouts {
+    create = "15m"
+    update = "15m"
+    delete = "15m"
+  }
+  parameters = {
```
Comment (hkantare) [inline website/docs/r/resource_instance.html.markdown:55]: parameters shouldnot have = it doesn't work from terraform v12 and higher version.....Terraform11 use to support this..

parameters {
------
}

File: website/docs/r/resource_instance.html.markdown [hunk 2]
```diff
@@ -35,6 +35,33 @@ resource "ibm_resource_instance" "resource_instance" {
 }
 ```
```
Comment (hkantare) [inline website/docs/r/resource_instance.html.markdown:37]: Can we have a title for this section in Bold something 
Provisioning a Hyper Protect DBaaS PostgreSQL Service...
something like this example 
https://github.com/IBM-Cloud/terraform-provider-ibm/blob/master/website/docs/r/iam_access_group_policy.html.markdown#access-group-policy-for-all-identity-and-access-enabled-services

Comment (PanLiyuan) [reply website/docs/r/resource_instance.html.markdown:37]: @hkantare I added a title. Please review.

**Comment by hkantare** [comment]
Detect secrets failed can you please validate and add false positive secrets

**Comment by PanLiyuan** [comment]
@hkantare I don't understand. Could you explain how I should do this?

**Comment by hkantare** [comment]
```
parameters = {
    "name": "cluster01",
    "admin_name": "admin",
    "password": "Hyperprotectdbaas0001"
    "confirm_password": "Hyperprotectdbaas0001",
    "db_version": "10"
  }
  ```
  
  From this section of code the detect_secret tool is detecting mostly the password as secret ..
  https://w3.ibm.com/w3publisher/detect-secrets/developer-tool
  Install the detect secret tool 
  detect-secrets scan --update .secrets.baseline
  detect-secrets audit .secrets.baseline (to mark it as false positive)
  push the updated secret base line also

**Comment by PanLiyuan** [comment]
@hkantare thanks a lot for your help, but I couldn't install Homebrew because of connection issues. I tried many times (maybe it's because I'm working from home). Is there any workaround for this secret problem?

**Comment by hkantare** [comment]
I raised a PR to your brach by updating detect secret ...
https://github.com/PanLiyuan/terraform-provider-ibm/pull/1

Can you merge this so that it will update this pr with required file

**Comment by PanLiyuan** [comment]
@hkantare thank you for your help! I merged that PR.

**Comment by hkantare** [comment]
There are still some conflicts ...so I raised a PR from my end
https://github.com/IBM-Cloud/terraform-provider-ibm/pull/2683
Can you check this so that we can merge this pr and close this pr

**Comment by hkantare** [comment]
closing this PR

**Pull request closed without merging**